plugins/tiddlywiki/multiwikiserver/modules/routes/handlers/post-bag.js

```diff
@@ -13,42 +13,70 @@ description
 \*/
 (function() {
 
-/*jslint node: true, browser: true */
-/*global $tw: false */
-"use strict";
-
-exports.method = "POST";
-
-exports.path = /^\/bags$/;
-
-exports.bodyFormat = "www-form-urlencoded";
-
-exports.csrfDisable = true;
-
-exports.useACL = true;
-
-exports.entityName = "bag"
-
-exports.handler = function(request,response,state) {
-	if(state.data.bag_name) {
-		const result = $tw.mws.store.createBag(state.data.bag_name,state.data.description);
-		if(!result) {
-			state.sendResponse(302,{
-				"Content-Type": "text/plain",
-				"Location": "/"
-			});
+	/*jslint node: true, browser: true */
+	/*global $tw: false */
+	"use strict";
```

Comment:
with TW do not indent the code inside the outer `(function() {` wrapper. -- It is intended to remove those wrapper functions in a future PR.

Reply:
As @pmario points out, it is important to match the formatting conventions used elsewhere in the code. I appreciate that that often means fighting tooling that wants to do things differently, and we do have an interest in improving the situation.